pyproject.toml

```diff
@@ -50,10 +50,20 @@ namespaces = false
 basic-pitch = "basic_pitch.predict:main"
 
 [project.optional-dependencies]
+train = [
+	"basic_pitch[tf,test]",
```

Comment:
```suggestion
	"basic_pitch[tf]",
```
If you want these in `test` then add them to test. Dont add test to train.

Reply:
would it make sense the other way around?

Reply:
of should these just be separated entirely? Is there a specific convention around sharing dependencies between optional lists?

Reply:
Not sure if there's a convention, but I wouldn't add test dependencies to anything that's meant to be installed by an end user.